### PR TITLE
changelog: update branch 2.4 `CHANGELOG.md` with missing entries

### PR DESCRIPTION
**Pull request by Ebaneck**
**Component**:

'changelog'

**Context**: 

Update changelog file with missing entries for branch 2.4.3

**Summary**:

**Acceptance criteria**: 

---

Closes: #ISSUE_NUMBER

### CODE REVIEW

**Review by alexandre-allard — COMMENTED** [review]
We need to at least add an entry for Solutions:
```
### Features added
- [#1993](https://github.com/scality/metalk8s/issues/1993) - Add Solutions management,
CLI tooling to deploy Solutions (complex Kubernetes applications)
(PR [#2279](https://github.com/scality/metalk8s/pull/2279)
```

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,40 @@
...
+(PR [#2248](https://github.com/scality/metalk8s/pull/2248))
+
+- Fix python invalid `time.sleep` argument
+(PR [#2257](https://github.com/scality/metalk8s/pull/2257))
+
+- Better handling of `wait_minions` when `test.ping` raises an exception
+(PR [#2308](https://github.com/scality/metalk8s/pull/2308))
+
+- Better handling of `wait_minions` when `test.ping` raises an exception
+(PR [#2308](https://github.com/scality/metalk8s/pull/2308))
```
Comment (gdemonet) [inline CHANGELOG.md]: I think if we don't have documented bugs, we don't need an item in the Changelog. Those three are "internal details", I would get rid of them.

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,40 @@
...
+
+### Documentation
+- Got rid of non-english word
+(PR [#2298](https://github.com/scality/metalk8s/pull/2298))
```
Comment (gdemonet) [inline CHANGELOG.md]: Same, no real impact for user considering to upgrade/install this version, so let's remove it :)

**Review by gdemonet — COMMENTED** [review]
Except for a few items that may be removed, this LGTM :) Thanks @Ebaneck

File: CHANGELOG.md (hunk 1 above)
Comment (Ebaneck) [reply CHANGELOG.md]: Done

File: CHANGELOG.md (hunk 2 above)
Comment (Ebaneck) [reply CHANGELOG.md]: Done

**Review by gdemonet — APPROVED** [review]

**Comment by TeddyAndrieux** [comment]
This PR only affect changelog and look like we have an issue on 2.6, let's merge this one as it also do not change anything on 2.6 branch

**Pull request merged**